lib/config/folderconfiguration.go

```diff
@@ -226,6 +226,12 @@ func (f *FolderConfiguration) prepare(myID protocol.DeviceID, existingDevices ma
 	} else if f.MaxConcurrentWrites > maxConcurrentWritesLimit {
 		f.MaxConcurrentWrites = maxConcurrentWritesLimit
 	}
+
+	if f.Type == FolderTypeReceiveEncrypted {
+		f.IgnorePerms = true
+		f.Versioning.Type = ""
+		f.Versioning.Params = nil
```

Comment:
Versioning I am not as certain any more. It should just work in principle when it comes to versioning items in the code. Recovering items will be somewhat painful to users: All the UI recovering is likely useless, and on disk won't be trivial either. At the same time it's possible (once the de-cryption tooling is there, and that will happen). And it's quite likely, that encrypted nodes will be used as central hubs, which makes them suitable for versioning (all changes there will be remote changes).

What are your thoughts on disabling versioning or not for receive-encrypted folders?

Reply:
I think we should default to it off, but not lock it down any further at this point.

Reply:
Sounds reasonable, but I can't oversee the consequences because of limited experience with how versioning really works.